content/webapp/components/EventSchedule/EventSchedule.tsx

```diff
@@ -19,30 +54,35 @@ const EventSchedule: FunctionComponent<Props> = ({ schedule }) => {
     .filter(({ isNotLinked }) => isNotLinked)
     .map(({ event }) => event.id);
 
+  // We split scheduled events into two headings: Events and Past Events.
+  //
+  // This is primarily for the benefit of long-running, repeating events
+  // like the Lights Up sessions for Milk.  There are lots of events in
```

Comment:
We could add [a link](https://wellcomecollection.org/events/ZCRYYRQAAB3ySUc2) for future reference?

Reply:
Good suggestion 👍